x-pack/legacy/plugins/integrations_manager/public/screens/detail/markdown_renderers.tsx

```diff
@@ -25,25 +23,6 @@ const REL_NOFOLLOW = 'nofollow';
 /** prevents the browser from sending the current address as referrer via the Referer HTTP header */
 const REL_NOREFERRER = 'noreferrer';
 
-export const WrappedEuiImage = ({
```

Comment:
We decided to just use images here as we don't need Figure and Figcaption and they are causing invalid html warnings due to being wrapped in paragraph tags.